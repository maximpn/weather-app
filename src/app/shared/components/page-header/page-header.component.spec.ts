import { Component } from '@angular/core';
import { ComponentFixture, TestBed } from '@angular/core/testing';

import { PageHeaderComponent } from './page-header.component';

@Component({
  template: '<wa-page-header><h1>Some header</h1></wa-page-header>',
})
class WrapperComponent {}

describe('ErrorGuardComponent', () => {
  let fixture: ComponentFixture<WrapperComponent>;

  beforeEach(() => {
    fixture = TestBed.configureTestingModule({
      declarations: [
        PageHeaderComponent,
        WrapperComponent,
      ],
    }).createComponent(WrapperComponent);

    fixture.detectChanges();
  });

  it('should render the component', () => {
    expect(fixture.nativeElement).toMatchSnapshot();
  });
});
