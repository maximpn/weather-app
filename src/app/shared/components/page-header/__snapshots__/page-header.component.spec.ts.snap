// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeaderComponent should render the component 1`] = `
<div
  id="root0"
>
  <wa-page-header>
    <header>
      <h1>
        Some header
      </h1>
    </header>
  </wa-page-header>
</div>
`;
